server/services/offlinePayment.service.js

```diff
@@ -42,17 +39,21 @@ export const addOfflinePayment = async (offlinePayment) => {
     );
   }
 
-  const property = await getPropertyById(offer.propertyId);
-
   try {
     const payment = await new OfflinePayment(offlinePayment).save();
-    const description = `You added an offline payment of ${getMoneyFormat(
-      payment.amount,
-    )} for ${getFormattedName(property.name)}`;
+
+    if (offlinePayment.model.type === VALID_TRANSACTION_MODELS.OFFER) {
```

Comment:
Since we are in the offline payment service, using `VALID_TRANSACTION_MODELS` looks awkward so it is better to use `MODEL`

```suggestion
    if (offlinePayment.model.type === MODEL.OFFER) {
```